mindjunkies/forums/models.py

```
@@ -45,17 +45,18 @@ class ForumReply(models.Model):
-    
+
     # Simple reaction counter
     like_count = models.PositiveIntegerField(default=0)
-    
+
     class Meta:
         ordering = ['created_at']
         verbose_name_plural = 'Forum replies'
-    
+
     def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)